src/components/dialogs/connectivity/connectivity-form-utils.ts

```diff
@@ -106,7 +107,7 @@ export const getConnectivityWithoutPositionEmptyFormData = (id = CONNECTIVITY) =
     [id]: getConnectivityPropertiesEmptyFormData(),
 });
 
-export const getConnectivityVoltageLevelData = ({ voltageLevelId }) => {
+export const getConnectivityVoltageLevelData = ({ voltageLevelId }: { voltageLevelId?: string | null }) => {
```

Comment:
```suggestion
export const getConnectivityVoltageLevelData = ({ voltageLevelId }: { voltageLevelId?: Input }) => {
```
you have to add import for Input type from commons-ui

```
import { Input } from '@gridsuite/commons-ui';
```